edx_sysadmin/views.py

```diff
@@ -149,6 +157,118 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
             )
 
 
+class GitImport(SysadminDashboardView):
+    """
+    This provide the view to load new course from github
+    """
+    
+    template_name = 'edx_sysadmin/gitimport.html'
+
+    def git_info_for_course(self, cdir):
```

Comment:
@umarmughal824 where is this method being used?

Reply:
@ziafazal bhai, it was used in showing the table of the courses but now we are not using it anymore as per @pdpinch suggestion. So removing that function for now.